xpra/server/auth/keycloak_auth.py

```diff
@@ -0,0 +1,125 @@
+# This file is part of Xpra.
+# Copyright (C) 2016-2021 Antoine Martin <antoine@xpra.org>
+# Copyright (C) 2022 Nathalie Casati <nat@yuka.ch>
+# Xpra is released under the terms of the GNU GPL v2, or, at your option, any
+# later version. See the file COPYING for details.
+
+import os
+import json
+
+from xpra.util import typedict
+from xpra.server.auth.sys_auth_base import SysAuthenticator, log
+from keycloak import KeycloakOpenID
+from oauthlib.oauth2 import WebApplicationClient
```

Comment:
Please move these two imports into the methods that uses them so that the module can be loaded and will only cause an `ImportError` if we actually try to use it and the libraries are missing.